.ci/oci-launch-e2e.sh

```diff
@@ -14,7 +14,7 @@ export ARTIFACTS_DIR=${ARTIFACT_DIR:-"/tmp/appstudio"}
 
 function executeE2ETests() {
     make build
-    "${WORKSPACE}"/bin/e2e-appstudio --ginkgo.junit-report="${ARTIFACTS_DIR}"/e2e-report.xml --ginkgo.progress --ginkgo.v
+    "${WORKSPACE}"/bin/e2e-appstudio --ginkgo.junit-report="${ARTIFACTS_DIR}"/e2e-report.xml --ginkgo.progress --ginkgo.v --ginkgo.label-filter='!ec
```

Comment:
Recently we started to use the label `Pending` for tests that are temporarily disabled (i.e. under development): for example see [jvm-build-service](https://github.com/redhat-appstudio/e2e-tests/blob/a74758abd0e657a15e70c4ad39f9aab1cf808081/tests/build/jvm-build.go#L37)

Could you please update ec tests similarly? Thanks

Reply:
The tests labeled `ec` are all expected to fail. The existing label is exactly what we want in this case.